tools/pip-requires

```diff
@@ -15,58 +15,57 @@
 #
 
 setuptools==20.9.0
-pbr>=1.6
-PyMySQL>=0.6.2,!=0.7.7 # MIT License
-SQLAlchemy>=1.0.10,<1.1.0  # MIT
-alembic>=0.8.0
+pbr>=0.6,<1.0
+PyMySQL>=0.6.2 # MIT License
+SQLAlchemy>=0.7.8,<=0.8.99
+alembic>=0.4.1
 amqplib>=0.6.1
 anyjson>=0.3.3
 argparse
-eventlet!=0.18.3,>=0.18.2  # MIT
-kombu>=3.0.25
+eventlet>=0.13.0
+kombu>=2.4.8
 lockfile>=0.8
 lxml>=2.3
 WebOb>=1.2.3
 greenlet==0.4.9
-futures>=3.0
-sqlalchemy-migrate>=0.9.6
-netaddr>=0.7.12,!=0.7.16  # BSD
-paramiko>=1.16.0
-iso8601>=0.1.9
+futures>=2.1.3
+sqlalchemy-migrate>=0.8.2,!=0.8.4
+netaddr>=0.7.6
+paramiko>=1.9.0
+iso8601>=0.1.8
 python-novaclient>=2
 python-neutronclient>=3
 pyvmomi>=6.0.0
-stevedore>=1.5.0
-websockify>=0.6.1
-oslo.rootwrap>=2.0.0 # Apache-2.0
-oslo.concurrency>=3.7.0
+stevedore>=0.14
+websockify>=0.5.1,<0.6
+oslo.rootwrap<1.7.0,>=1.6.0 # Apache-2.0
+oslo.concurrency>=3.0.0
 oslo.config>=3.0.0
-oslo.context>=0.2.0
-oslo.log>=1.14.0
-oslo.messaging>=4.0.0,<=4.6.1 # Apache-2.0
+oslo.context==1.0.0
+oslo.log==2.0.0
+oslo.messaging>=4.0.0 # Apache-2.0
 oslo.versionedobjects>=1.4.0 # Apache-2.0
-oslo.service>=1.0.0
-oslo.utils>=3.5.0
+oslo.service==1.0.0
+oslo.utils==3.0.0
 oslo.db>=4.1.0
 oslo.vmware>=1.16.0
-pecan>=1.0.0
-six>=1.9.0
+pecan>=0.4.5
+six>=1.5.2
 jsonpatch>=1.1
-WSME>=0.8
+WSME>=0.6
 Jinja2
 python-seamicroclient>=0.1.0,<2.0
 PasteDeploy>=1.5.0
 routes>=1.12.3
 httplib2>=0.7.5
 pycrypto>=2.6
 paste>=1.7.4
-passlib>=1.6
-pyOpenSSL>=0.14
+passlib>=1.5.3
+pyOpenSSL>=0.13
 suds==0.4
-Babel>=1.3,!=2.3.0,!=2.3.1,!=2.3.2,!=2.3.3  # BSD
+Babel==1.3
 posix_ipc
-pycadf>=1.1.0,!=2.0.0  # Apache-2.0
+pycadf>=0.4.1,<1.0.0
 ansible==1.9.4
 configparser
 pywinrm==0.1.1
-keystonemiddleware==2.3.4
```

Comment:
Keystonemiddleware is required.
UTs are failing without keystonemiddleware

======================================================================
ERROR: Failure: ImportError (No module named keystonemiddleware)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jenkins/sagar/hpe-eon/.venv/local/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/home/jenkins/sagar/hpe-eon/.venv/local/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/home/jenkins/sagar/hpe-eon/.venv/local/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/home/jenkins/sagar/hpe-eon/eon/api/acl.py", line 21, in <module>
    from keystonemiddleware import auth_token as keystone_auth_token
ImportError: No module named keystonemiddleware

=======================================================

Also EON functionality requires keystonemiddleware

After adding that package, EON UTs passed

eon/virt/vmware/hlm_input_model.py                                     71      4    94%
eon/virt/vmware/inventory_collector.py                                218     30    86%
eon/virt/vmware/network_prop_json_handler.py                          321     14    96%
eon/virt/vmware/utils.py                                               79     28    65%
eon/virt/vmware/validator.py                                           41      2    95%
eon/virt/vmware/vcenter_manager.py                                    171     28    84%
eon/virt/vmware/vim_util.py                                            42      3    93%
---------------------------------------------------------------------------------------
TOTAL                                                               15436   6293    59%
----------------------------------------------------------------------
Ran 682 tests in 33.004s

OK